apps/asap-server/scripts/create-migration.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+input=$1
+
+if [ -z "$input" ]; then
+    echo "Missing argument - migration name"
+    exit
+fi
+
+timestamp=$(date +%s%N | cut -b1-13)
```

Comment:
```suggestion
timestamp=$(date +%s)
```
Since this is epoch time, we don't need to worry about cutting the date shorter.
The `%N` wasn't doing anything both on my mac and on linux busybox docker img. Am I doing smth wrong?

Reply:
converts nano to mili:
![image](https://user-images.githubusercontent.com/11645956/114185848-14bec200-993e-11eb-969e-f31fea4e9842.png)